server/lib/get-word-count.js

```diff
@@ -2,8 +2,8 @@
 
 const RE_VALID_ONE_CHARS = /^[IAa0-9]$/;
 
```

Comment:
Given that in that in another PR your replaces the home brew function, is this file actually doing anything?